.evergreen/csfle/start-servers.sh

```diff
@@ -30,50 +30,60 @@ done
 . ./activate-kmstlsvenv.sh
 
 # The -u options forces the stdout and stderr streams to be unbuffered.
-# TMPDIR is required to avoid "AF_UNIX path too long" errors.
+COMMAND="python -u"
+if [ "$(uname -s)" != "Darwin" ]; then
+  # On linux and windows host, we need to use nohup to daemonize the process
+  # and prevent the task from hanging.
+  # The macos hosts do not support nohup.
```

Comment:
What happens on macos?

Reply:
It does not hang on macos, as confirmed by the test run: https://spruce.mongodb.com/task/drivers_tools_tests_all__os_fully_featured~macos_14_auth~auth_ssl~nossl_test_csfle_patch_eadf5c21e411d5582e52b43cfe9e2ffd26274fb8_677beac5430c5d0007e4745d_25_01_06_14_37_57/logs?execution=0&sortBy=STATUS&sortDir=ASC.